mars/tensor/reduction/tests/test_reduction_execute.py

```diff
@@ -173,6 +173,12 @@ def testVarExecution(self):
         raw1 = np.random.random((20, 25))
         raw2 = np.random.randint(10, size=(20, 25))
 
+        arr0 = tensor(raw1)
```

Comment:
Better to specify `chunk_size` to a number, prevent from possible intelligent chunk size even though the tensor size is small.